packages/report-server/examples.http

```diff
@@ -12,7 +12,7 @@ content-type: {{contentType}}
 Authorization: {{authorization}}
 
 ### Fetch Report With Test Config
-POST {{baseUrl}}/testFetchReport/xx?organisationUnitCodes=TO&hierarchy=psss&period=2020 HTTP/1.1
+POST {{baseUrl}}/testReport?organisationUnitCodes=TO&hierarchy=psss&period=2020 HTTP/1.1
```

Comment:
Slight url renaming here, and I'm glad to be done away with the need to specify a dummy report code 👍